app/models/review.rb

```diff
@@ -29,6 +29,13 @@ def get_score
     total
   end
 
+  def self.order_by_vote
+    select('reviews.*, COALESCE(sum(votes.value), 0) AS score')
+    .joins('LEFT JOIN votes ON votes.review_id = reviews.id')
+    .group("reviews.id")
+    .order("score desc")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.